### ISSUE TITLE
Make the creating user the default administrator and store 'created_by" in a business

### ISSUE THREAD

**Issue by vikasagarwal** [OrderMail/blacklabel#25]
###Acceptance Criteria###

1. When a user creates a business, he/she should become an administrator automatically. 
2. A newly created business should also store the record of who created the business. This is one-time activity per business and never changes.

**Comment by priyankshukla** [comment]
I believe that has covered in #16 story.
When a user creates a business, he/she gets enrolled as CreatedBy in business schema/document.

**Comment by vikasagarwal** [comment]
Does it mean that the user also has become the administrator? Or, are you
confusing created_by with isAdmin?

On Fri, Mar 20, 2015 at 8:17 AM, Priyank Shukla <notifications@github.com>
wrote:

> I believe that has covered in #16
> <https://github.com/OrderMail/blacklabel/issues/16> story.
> When a user creates a business, he/she gets enrolled as CreatedBy in
> business schema/document.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OrderMail/blacklabel/issues/25#issuecomment-83864854>.
>



-- 
Best Cricket T20 Coverage on:
http://cricket-t20.com

**Comment by priyankshukla** [comment]
No, isAdmin and createdBy are two different things. The one who introduces the business and the other who is controlling the business account in blacklabel.

My Understanding-
A person, who is creating a business, will become a admin(by default) for that business. That might get change later on once we introduce the functionality to set the Admin separately.

Please correct if I am wrong here.
